homeassistant/components/shelly/strings.json

```diff
@@ -160,6 +160,14 @@
     "push_update_failure": {
       "title": "Shelly device {device_name} push update failure",
       "description": "Home Assistant is not receiving push updates from the Shelly device {device_name} with IP address {ip_address}. Check the CoIoT configuration in the web panel of the device and your network configuration."
+    },
+    "deprecated_valve_switch": {
+      "title": "The switch entity for Shelly Gas Valve is deprecated",
+      "description": "The switch entity for Shelly Gas Valve is deprecated, a valve entity is available and should be used going forward."
```

Comment:
```suggestion
      "description": "The switch entity for Shelly Gas Valve is deprecated. A valve entity is available and should be used going forward."
```

Reply:
Would it be possible to include which valve entity should be used instead and what valve services should be used instead?